<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ØªÙØ§ØµÙŠÙ„ Ø§Ù„Ù‚Ø·Ø¹Ø©</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget { font-family: 'Leelawadee UI', sans-serif; background-color: #f4f7f6; }
    
    #lblTitle { font-size: 26px; font-weight: bold; color: #2c3e50; margin-bottom: 10px; }
    
    /* Ø§Ù„Ø¹Ù†Ø§ÙˆÙŠÙ† Ø§Ù„Ø¬Ø§Ù†Ø¨ÙŠØ© */
    #l_inv, #l1, #l2, #l3, #l4, #l5, #l6, #l7, #l8, #l9, #l10, #l_src, #l_dims, #l_wgt, #l_editor, #l_edate { 
        font-weight: bold; color: #7f8c8d; font-size: 14px; 
    }
    
    /* Ù‚ÙŠÙ… Ø§Ù„Ø¨ÙŠØ§Ù†Ø§Øª */
    #valCode, #valName, #valSource, #valType, #valQuantity, #valMaterial, #valDimensions, #valWeight, 
    #valPeriod, #valCondition, #valStorage, #valDescription, #valEditor, #valEditDate { 
        background-color: white; 
        border: 1px solid #ddd; 
        border-radius: 8px; 
        padding: 10px; 
        color: #2c3e50; 
        font-size: 15px; 
        font-weight: 500; 
    }
    
    #valInventory { 
        background-color: #e3f2fd; 
        color: #1565c0; 
        border: 1px solid #90caf9; 
        font-weight: bold; 
        padding: 10px; 
        border-radius: 8px;
    }
    #valNotes { 
        background-color: #fff3cd; 
        border: 1px solid #ffeeba; 
        color: #856404; 
        padding: 10px; 
        border-radius: 8px;
    }
    
    #imageFrame { background-color: white; border: 1px solid #ddd; border-radius: 12px; }
    #lblImage { background-color: #333; border-radius: 8px; color: white; }
    
    #btnNext, #btnPrev { background-color: rgba(0,0,0,0.6); color: white; border-radius: 20px; font-weight: bold; font-size: 18px; }
    #btnNext:hover, #btnPrev:hover { background-color: rgba(0,0,0,0.8); }
    
    #btnBack { background-color: #95a5a6; color: white; border-radius: 8px; padding: 8px 20px; font-weight: bold; border: none; }
    #btnEdit { background-color: #f39c12; color: white; border-radius: 8px; padding: 8px 20px; font-weight: bold; border: none; }
    #btnDelete { background-color: #e74c3c; color: white; border-radius: 8px; padding: 8px 20px; font-weight: bold; border: none; }
   </string>
  </property>
  
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing"> <number>20</number> </property>
   <property name="margin"> <number>30</number> </property>
   
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item> <widget class="QLabel" name="lblTitle"> <property name="text"> <string>ØªÙØ§ØµÙŠÙ„ Ø§Ù„Ù‚Ø·Ø¹Ø© Ø§Ù„Ø£Ø«Ø±ÙŠØ©</string> </property> </widget> </item>
     <item> <spacer name="spacer"> <property name="orientation"> <enum>Qt::Horizontal</enum> </property> </spacer> </item>
     <item> <widget class="QPushButton" name="btnBack"> <property name="text"> <string>Ø¹ÙˆØ¯Ø©</string> </property> </widget> </item>
    </layout>
   </item>

   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <property name="spacing"> <number>30</number> </property>
     
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape"> <enum>QFrame::NoFrame</enum> </property>
       <property name="widgetResizable"> <bool>true</bool> </property>
       <widget class="QWidget" name="scrollContents">
        <layout class="QFormLayout" name="formLayout">
         <property name="spacing"> <number>15</number> </property>
         
         <item row="0" column="0"> <widget class="QLabel" name="l_inv"> <property name="text"> <string>Ø±Ù‚Ù… Ø§Ù„Ø¬Ø±Ø¯:</string> </property> </widget> </item> 
         <item row="0" column="1"> <widget class="QLabel" name="valInventory"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="1" column="0"> <widget class="QLabel" name="l1"> <property name="text"> <string>Ø§Ù„ÙƒÙˆØ¯ Ø§Ù„Ø¢Ù„ÙŠ:</string> </property> </widget> </item> 
         <item row="1" column="1"> <widget class="QLabel" name="valCode"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="2" column="0"> <widget class="QLabel" name="l2"> <property name="text"> <string>Ø§Ù„Ø§Ø³Ù…:</string> </property> </widget> </item> 
         <item row="2" column="1"> <widget class="QLabel" name="valName"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="3" column="0"> <widget class="QLabel" name="l_src"> <property name="text"> <string>Ø§Ù„Ù…ØµØ¯Ø±:</string> </property> </widget> </item> 
         <item row="3" column="1"> <widget class="QLabel" name="valSource"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="4" column="0"> <widget class="QLabel" name="l3"> <property name="text"> <string>Ø§Ù„Ù†ÙˆØ¹:</string> </property> </widget> </item> 
         <item row="4" column="1"> <widget class="QLabel" name="valType"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="5" column="0"> <widget class="QLabel" name="l4"> <property name="text"> <string>Ø§Ù„Ø¹Ø¯Ø¯:</string> </property> </widget> </item> 
         <item row="5" column="1"> <widget class="QLabel" name="valQuantity"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="6" column="0"> <widget class="QLabel" name="l5"> <property name="text"> <string>Ø§Ù„Ù…Ø§Ø¯Ø©:</string> </property> </widget> </item> 
         <item row="6" column="1"> <widget class="QLabel" name="valMaterial"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="7" column="0"> <widget class="QLabel" name="l_dims"> <property name="text"> <string>Ø§Ù„Ù‚ÙŠØ§Ø³Ø§Øª:</string> </property> </widget> </item> 
         <item row="7" column="1"> <widget class="QLabel" name="valDimensions"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="8" column="0"> <widget class="QLabel" name="l_wgt"> <property name="text"> <string>Ø§Ù„ÙˆØ²Ù†:</string> </property> </widget> </item> 
         <item row="8" column="1"> <widget class="QLabel" name="valWeight"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="9" column="0"> <widget class="QLabel" name="l6"> <property name="text"> <string>Ø§Ù„ÙØªØ±Ø©:</string> </property> </widget> </item> 
         <item row="9" column="1"> <widget class="QLabel" name="valPeriod"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="10" column="0"> <widget class="QLabel" name="l7"> <property name="text"> <string>Ø§Ù„Ø­Ø§Ù„Ø©:</string> </property> </widget> </item> 
         <item row="10" column="1"> <widget class="QLabel" name="valCondition"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="11" column="0"> <widget class="QLabel" name="l8"> <property name="text"> <string>Ø§Ù„Ù…ÙˆÙ‚Ø¹:</string> </property> </widget> </item> 
         <item row="11" column="1"> <widget class="QLabel" name="valStorage"> <property name="text"> <string>---</string> </property> </widget> </item>
         
         <item row="12" column="0"> <widget class="QLabel" name="l9"> <property name="text"> <string>Ø§Ù„ÙˆØµÙ:</string> </property> </widget> </item> 
         <item row="12" column="1"> <widget class="QLabel" name="valDescription"> <property name="wordWrap"> <bool>true</bool> </property> <property name="text"> <string>---</string> </property> </widget> </item>
         
         <item row="13" column="0"> <widget class="QLabel" name="l10"> <property name="text"> <string>Ù…Ù„Ø§Ø­Ø¸Ø§Øª:</string> </property> </widget> </item> 
         <item row="13" column="1"> <widget class="QLabel" name="valNotes"> <property name="wordWrap"> <bool>true</bool> </property> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="14" column="0"> <widget class="QLabel" name="l_editor"> <property name="text"> <string>Ù…Ø­Ø±Ø± Ø§Ù„Ø¨Ø·Ø§Ù‚Ø©:</string> </property> </widget> </item> 
         <item row="14" column="1"> <widget class="QLabel" name="valEditor"> <property name="text"> <string>---</string> </property> </widget> </item>

         <item row="15" column="0"> <widget class="QLabel" name="l_edate"> <property name="text"> <string>ØªØ§Ø±ÙŠØ® Ø§Ù„ØªØ­Ø±ÙŠØ±:</string> </property> </widget> </item> 
         <item row="15" column="1"> <widget class="QLabel" name="valEditDate"> <property name="text"> <string>---</string> </property> </widget> </item>

        </layout>
       </widget>
      </widget>
     </item>

     <item>
      <widget class="QFrame" name="imageFrame">
       <property name="minimumSize"> <size> <width>500</width> <height>0</height> </size> </property>
       <layout class="QVBoxLayout" name="vImg">
        <item> 
         <widget class="QLabel" name="lblImage"> 
          <property name="minimumSize"> <size> <width>480</width> <height>320</height> </size> </property>
          <property name="alignment"> <set>Qt::AlignCenter</set> </property> 
          <property name="text"> <string>Ù„Ø§ ØªÙˆØ¬Ø¯ ØµÙˆØ±</string> </property> 
          <property name="scaledContents"> <bool>true</bool> </property>
         </widget> 
        </item>
        <item>
         <layout class="QHBoxLayout" name="hControls">
          <item> <widget class="QPushButton" name="btnNext"> <property name="text"> <string>â–¶</string> </property> <property name="fixedSize"> <size> <width>40</width> <height>40</height> </size> </property> </widget> </item>
          <item> <widget class="QLabel" name="lblImageCounter"> <property name="alignment"> <set>Qt::AlignCenter</set> </property> <property name="text"> <string>0 / 0</string> </property> </widget> </item>
          <item> <widget class="QPushButton" name="btnPrev"> <property name="text"> <string>â—€</string> </property> <property name="fixedSize"> <size> <width>40</width> <height>40</height> </size> </property> </widget> </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>

   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Footer">
     <item> <spacer name="horizontalSpacer_2"> <property name="orientation"> <enum>Qt::Horizontal</enum> </property> </spacer> </item>
     <item> <widget class="QPushButton" name="btnDelete"> <property name="text"> <string>ğŸ—‘ï¸ Ø­Ø°Ù Ø§Ù„Ù‚Ø·Ø¹Ø©</string> </property> </widget> </item>
     <item> <widget class="QPushButton" name="btnEdit"> <property name="text"> <string>âœï¸ ØªØ¹Ø¯ÙŠÙ„ Ø§Ù„Ø¨ÙŠØ§Ù†Ø§Øª</string> </property> </widget> </item>
    </layout>
   </item>
  </layout>
 </widget>
</ui>
